zegami_sdk/collection.py

```diff
@@ -1020,6 +1025,35 @@ def __len__(self) -> int:
     def __repr__(self) -> str:
         return "<Collection id={} name={}>".format(self.id, self.name)
 
+    def id_to_class(self, ID):
+        return self.classes[ID - 1]['name']
+
+    def get_annotations_metadata_to_csv(self, outpath='output.csv',
+                                        anno_type=None) -> pd.DataFrame:
+        """Creates a .csv file with annotations metadata from a collection.
+        Defaults to searching for annotations of all types.
+        """
+        metadata = self.get_annotations_metadata(anno_type)
+        metadata.to_csv(outpath)
```

Comment:
Unless you want to keep the DataFrame's index column, I usually add `index=False` to `DataFrame.to_csv` to avoid an unexpected column.